subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/execution/Interpreter.kt

```diff
@@ -67,7 +67,6 @@ import java.lang.reflect.InvocationTargetException
  * @see ResidualProgram
  * @see ResidualProgramCompiler
  */
-internal
```

Comment:
❓ any chance there are now public members that could be made `internal` to limit the surface?

Reply:
`ProgramId` exposes no members and `Interpreter` exposes only `eval` and the `Host` interface, both required by the testing infrastructure.